app/javascript/components/recipes/new/search-ingredient-list.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="w-96">
+    <div class="flex space-x-4">
+      <input
+        :placeholder="$t(`msg.ingredients.search`)"
+        class="w-full mx-1 my-4 p-2 border-2 rounded border-gray-300"
+      />
+    </div>
+    <table class="min-w-full divide-y divide-gray-200 border-2">
+      <thead class="justify-between">
+        <tr class="bg-gray-100  py-1">
+          <th 
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Trailing spaces not allowed                                                  no-trailing-spaces